app/controllers/issues_controller.rb

```diff
@@ -121,6 +123,13 @@ def import
   end
 
   private
+  def all_evidence_columns
+    @issue.evidence
+          .map { |evidence| evidence.fields.keys - ['Title', 'Label']  }
+          .flatten
+          .uniq
+  end 
```

Comment:
This is quacking like a setter (set_columns?) that needs to go into Issues::EvidenceController

From there we can also make the call of doing SQL eager loading to avoid N+1 in the query.